### DISCUSSION TITLE
할말 있으면 여기에 다셈

### DISCUSSION THREAD

**Discussion by keisiki** [keisiki/TWKR-Textbung#21] [Ideas]
안 까먹고 일하게 하고 싶다면

**Comment by Evendawn** [comment]
(1+딜 배수)^2=(370*레벨차)^3 이런 식으로 사칙연산 이외의 계산도 가능함? 가능하면 저거 적용하고 -1제곱 먹여서 역수뽑아서 레벨 낮은쪽한테 데미지 배수로 적용하면 될것같은데
저걸 그대로 적용하면 100레벨 차이에서 대략 4배의 딜뻥 및 뎀감을 보이게 될거임

**Reply by keisiki** [reply]
상관없는데 저기서 -1제곱 먹이면 뭐가되더라...기억이 안나네

**Reply by Evendawn** [reply]
역수가 나오지

**Reply by keisiki** [reply]
위에껀 가능한데 역수를 만드는건 좀 힘들지도

**Reply by Evendawn** [reply]
아니면 생각해보니깐 저것도 밸런스 더럽게 안맞는 것 같아서 50레벨 이상 차이날때만, 레벨차-50 해서 적용하는게 나을지도
따로 전용 메세지 뜨게 하고 적용하고 나서 신키도 레벨 600 초중반대로 올리고

**Reply by keisiki** [reply]
> 역수가 나오지

정수의 역수면 분수가 되는거잖아? 그러면 그 값은 깨져버려, 저 값 자체를 분모 변수에 할당하고 그걸 대입해 나누는건 할수 있긴함

**Reply by Evendawn** [reply]
부동소수점을 안쓰나보네? 그러면 위에서 생각해놓은 저 안으로, 날먹방지만 하게 플레이어쪽에서는 적용 안되도록 하고 적용하면 되겠다

**Reply by keisiki** [reply]
근데 (1+딜 배수)^2=(370*레벨차)^3 
이면 어떻게 적용하는거임?
저 딜 배수 만큼 데미지 증가?

**Reply by Evendawn** [reply]
예를들어 x^2=370y^3에서 x=100이면 y가 대략 3 근처가 나오니깐 실제 적용 데미지가 4배로 들어가는거지
그와중에 딜배수랑 레벨차랑 위치 바꿔서 썼네

**Reply by Evendawn** [reply]
고정소수점은 사용할 수 있으니깐 대충 소숫점 2자리 또는 3자리에서 반올림하면 되지 않을까

**Reply by keisiki** [reply]
고정소수점은 되나? 안되면 100배하고 쓰면되긴하겠네

**Reply by keisiki** [reply]
즉,  (1+레벨차)^2=(370*딜배수)^3 란 거지?

**Reply by Evendawn** [reply]
370x^3=y^2
여기서 x는 딜 배수, y는 레벨차-50
나온 딜 배수값에 1 더해서 데미지 계산할때 이것만큼 곱하면 될거야

**Reply by keisiki** [reply]
power(레벨차,2) = power(370*(배율), 3)
인데 이걸 활용하려면 이 식을 (배율)에 대한 식으로 고쳐야하는데...에라위키를 뒤져봐도 세제곱근을 표현하는 식을 못찾겠네

**Comment by Evendawn** [comment]
1. 죽인 상대 혼에서 습득하는 경험치량을 전반적으로 너프
2. 한 회차에서 얻을 수 있는 경험치량의 제한
3. 한 상대에서 얻을 수 있는 경험치량의 제한
4. 1+2 또는 1+3
이렇게 4개 안 중에서 뭐가 나을 것 같음

**Reply by Evendawn** [reply]
설마 텍붕이모드를 하루에 몇 시간씩 하는 피학성애자가 나올줄은 몰랐으나 그 피학성애자가 보여준 결과를 봐선 경험치 습득량을 너프해야할 것 같은데

**Reply by keisiki** [reply]
1+3이 나을거 같은데

**Comment by magusviolet** [comment]
아카에 버그제보로 쓰면 사람들이 악용할까봐 여기다 담
요리를 만들때 이름을 '생명력/신력' 이렇게 달아버리면 해당 맛이 있는걸로 판정해서 특수효과가 발동함.
여러 맛을 발동시키는 것도 가능했고

**Reply by keisiki** [reply]
https://github.com/keisiki/TWKR-Textbung/commit/a710876eacd8796816438a80f6f705bae4b92175
수정해슴

**Reply by keisiki** [reply]
이름 하나로 소질넣는걸 생각 못했네
